### PR TITLE
Add basic apphosting.yaml in next.js and angular starter templates.

### PR DESCRIPTION
**Pull request by taeold**

### CODE REVIEW

**Review by Yuangwang — APPROVED** [review]

File: starters/angular/basic/apphosting.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Settings for Backend (on Cloud Run).
+# See https://firebase.google.com/docs/app-hosting/configure#cloud-run.
+runConfig:
+  minInstances: 0
+  # maxInstances: 100
+  # concurrency: 100
+  # cpu: 2
```
Comment (inlined) [inline starters/angular/basic/apphosting.yaml:7]: Should these values be the defaults?

File: starters/angular/basic/apphosting.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Settings for Backend (on Cloud Run).
```
Comment (inlined) [inline starters/angular/basic/apphosting.yaml:1]: Maybe we should have a header explaining the use of environment-specific apphosting.yaml overrides?

File: starters/nextjs/basic/apphosting.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Settings for Backend (on Cloud Run).
+# See https://firebase.google.com/docs/app-hosting/configure#cloud-run.
+runConfig:
+  minInstances: 0
```
Comment (inlined) [inline starters/nextjs/basic/apphosting.yaml:4]: why does Angular comment out all values but Nex.tjs does not?

Comment (inlined) [reply starters/nextjs/basic/apphosting.yaml:4]: I see that you use the variable in Next.js, but then why not use it in Angular SSR?

Comment (taeold) [reply starters/nextjs/basic/apphosting.yaml:4]: I have no idea how to use env vars in angular. If you do, welcoming a PR!

File: starters/angular/basic/apphosting.yaml (hunk 2 above)
Comment (taeold) [reply starters/angular/basic/apphosting.yaml:1]: I think I'd rather point the users to public doc so we don't have 2 separate places to track documentations on how to use env specific overrides.

File: starters/angular/basic/apphosting.yaml (hunk 1 above)
Comment (taeold) [reply starters/angular/basic/apphosting.yaml:7]: changed.

File: starters/nextjs/basic/apphosting.yaml (hunk 3 above)
Comment (inlined) [reply starters/nextjs/basic/apphosting.yaml:4]: process.env doesn't work?

Comment (taeold) [reply starters/nextjs/basic/apphosting.yaml:4]: no

**Pull request merged**